.github/release-drafter.yml

```diff
@@ -0,0 +1,54 @@
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'kind/enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'kind/bug'
+  - title: '🧰 Maintenance'
+    labels:
+      - 'kind/chore'
+      - 'area/dependencies'
+
+exclude-labels:
+  - duplicate
+  - invalid
+  - later
+  - wontfix
+  - question
+  - skip-changelog
+
+change-template: '- $TITLE (#$NUMBER)'
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
+name-template: 'Draft'
+template: |
+  $CHANGES
+
+autolabeler:
+    # Tag any PR with "!" in the subject as major update. In other words, breaking change
+  - label: 'major'
+    title: '/.*!:.*/'
```

Comment:
@viccuad, I've added this configuration to tag any PR with `!` in the title subject/scope as  major update.

Reply:
perfect! many thanks! :green_heart:

Reply:
@viccuad, please double check the other autolabeler rules for the other types of changes.